### PR TITLE
fix(clerk-js): Add a dummy base when constructing the url in isRedirectForFAPIInitiatedFlow

### PR DESCRIPTION
**Pull request by desiprisg**
## Type of change

- [x] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

## Packages affected

- [x] `@clerk/clerk-js`
- [ ] `@clerk/clerk-react`
- [ ] `@clerk/nextjs`
- [ ] `@clerk/remix`
- [ ] `@clerk/types`
- [ ] `@clerk/themes`
- [ ] `@clerk/localizations`
- [ ] `@clerk/clerk-expo`
- [ ] `@clerk/backend`
- [ ] `@clerk/clerk-sdk-node`
- [ ] `@clerk/shared`
- [ ] `@clerk/fastify`
- [ ] `@clerk/chrome-extension`
- [ ] `gatsby-plugin-clerk`
- [ ] `build/tooling/chore`

## Description

- [x] `npm test` runs as expected.
- [x] `npm run build` runs as expected.

### CODE REVIEW

File: packages/clerk-js/src/utils/url.ts [hunk 1]
```diff
@@ -362,3 +362,16 @@ export const mergeFragmentIntoUrl = (_url: string | URL): URL => {
 export const pathFromFullPath = (fullPath: string) => {
   return fullPath.replace(/CLERK-ROUTER\/(.*?)\//, '');
 };
+
+const frontendApiRedirectPaths: string[] = [
+  '/oauth/authorize', // OAuth2 identify provider flow
+  '/v1/verify', // magic links
+  '/v1/tickets/accept', // ticket flow
+];
```
Comment (dimkl) [inline packages/clerk-js/src/utils/url.ts:370]: @mzhong9723 should we also add `/choose` or other redirect urls? What urls should be contained here?

**Review by dimkl — APPROVED** [review]

File: packages/clerk-js/src/utils/url.ts (hunk 1 above)
Comment (SokratisVidros) [reply packages/clerk-js/src/utils/url.ts:370]: These are only FAPI paths for FAPI initiated redirection flows

**Review by SokratisVidros — APPROVED** [review]

File: packages/clerk-js/src/utils/url.ts [hunk 2]
```diff
@@ -362,3 +362,16 @@ export const mergeFragmentIntoUrl = (_url: string | URL): URL => {
...
+
+export function isRedirectForFAPIInitiatedFlow(clerk: Clerk, redirectUrl: string): boolean {
```
Comment (SokratisVidros) [inline packages/clerk-js/src/utils/url.ts:372]: @desiprisg Please unit test this.

Comment (desiprisg) [reply packages/clerk-js/src/utils/url.ts]: Done

**Pull request merged**